### PR TITLE
177-update-to-expo-sdk-50

### PR DESCRIPTION
**Pull request by alvrba**

### CODE REVIEW

File: src/components/AddItem.tsx [hunk 1]
```diff
@@ -52,6 +53,7 @@ const AddItem: FC<AddItemProps> = ({ parentId, refresh }) => {
   }>({
     toggle: false,
   });
+  const reducedMotion = useReducedMotion();
```
Comment (pyphilia) [inline src/components/AddItem.tsx:56]: What does it do? Please add some comments.

**Review by pyphilia — APPROVED** [review]
LGTM, but could you add some details here and/or in the PR to why you started to use `useReducedMotion`?

**Comment by alvrba** [comment]
@pyphilia I changed the condition of navigating to the main screen from the sign-in screen using the `userToken` instead of the `currentMember`. It was the condition used from the beginning and I believe it is more reliable and avoids the intermittent error of not navigating to the main screen when logging in.

**Review by spaenleh — APPROVED** [review]
Looks good to me !

**Pull request merged**